chat_shell/chat_shell/prompts/knowledge_base.py

```
@@ -13,4 +13,4 @@
 __all__ = [
     "KB_PROMPT_STRICT",
     "KB_PROMPT_RELAXED",
-]
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)